pmm/v3/pmm3-submodules.groovy

```diff
@@ -27,29 +27,42 @@ pipeline {
         label 'agent-amd64'
     }
     parameters {
-        choice(
-            choices: ['el9', 'next'],
-            description: 'The base OS of the image',
-            name: 'BASE_OS'
-        )
+        string(
+            defaultValue: 'v3',
+            description: 'Tag/Branch for pmm-submodules repository',
+            name: 'PMM_BRANCH')
+        string(
+            defaultValue: '',
+            description: 'URL for pmm-submodules repository PR',
+            name: 'CHANGE_URL')
+        string(
+            defaultValue: '',
+            description: 'ID for pmm-submodules repository PR',
+            name: 'CHANGE_ID')
+        string(
+            defaultValue: 'PR-2345',
+            description: 'Change Request Number for pmm-submodules repository PR',
+            name: 'BRANCH_NAME')
+    }
+
     }
     environment {
         PATH_TO_SCRIPTS = 'sources/pmm/src/github.com/percona/pmm/build/scripts'
-        PMM_VER =  """${sh(returnStdout: true, script: "cat VERSION").trim()}"""
     }
     stages {
         stage('Prepare') {
-            when {
-                beforeAgent true
-                expression { env.PMM_VER =~ '^3.' }
-            }
             steps {
+                git poll: false,
+                    branch: PMM_BRANCH,
+                    url: 'http://github.com/Percona-Lab/pmm-submodules'
                 withCredentials([string(credentialsId: 'GITHUB_API_TOKEN', variable: 'GITHUB_API_TOKEN')]) {
                 sh '''
                     set -o errexit
                     sudo rm -rf results tmp || :
                     git reset --hard
                     git clean -fdx
+                    git submodule update --init --jobs 10
+                    git submodule status
```

Comment:
Same: lines 61-63 are unnecessary on a fresh repo clone, so let's remove them.

This change is legit, however we need to remove the same from [line 80](https://github.com/Percona-Lab/jenkins-pipelines/pull/2473/files#diff-64a4dafc44f283828108478c168c8c4b933d7282c7d2324aa4a35fd0659739c7R80) to avoid duplication.